pyphi/repertoire.py

```diff
@@ -66,6 +52,7 @@ def forward_cause_probability(
         mechanism=purview,
         purview=mechanism,
         mechanism_state=purview_state,
+        use_backward_tpm=True
```

Comment:
May make sense to call this `direction` and have it accept a `direction.Direction`, to keep things consistent with the rest of the API.